### DISCUSSION TITLE
Select multiple layers at once

### DISCUSSION THREAD

**Discussion by MoRadwan74** [TerriaJS/terriajs#5244] [Q&A] [answered]
In TerriaJS, we can add any type of layer in a group or standalone to the catalog to be able to show them on the map. So, is there a way to select multiple layers at once or in other words, select a group and when selected, show its children layers?

**Comment by KeyboardSounds** [comment, marked as answer]
Hey @MoRadwan74,

We don't support this right now, but it would be a neat feature. Would you be able to create an Issue describing step by step how you picture this would work? Unfortunately our team are totally flat out busy on other work at the moment, so it's unlikely that we'll be able to implement it in the near future, but the more we know about how people want to use TerriaJS, the better we can make it.
